src/main/java/com/autonomic/tmc/auth/Token.java

```diff
@@ -88,4 +87,8 @@ boolean isExpired() {
             .isAfter(expiration);
     }
 
+    @Override
+    public String get() {
```

Comment:
If replacing Lombok, this should be getValue.

Reply:
👍  Good Catch